web/templates/shared/navigation.html.eex

```diff
@@ -1,54 +1,67 @@
-<header>
-  <div class="o-grid o-grid--wrap">
-    <div class="o-grid__cell o-grid__cell--width-100 o-grid__cell--width-50@medium">
-      <%= if assigns[:logo] == nil || assigns[:logo] == true do %>
-        <a href="/"><img src='/images/logo.svg' alt="<%= gettext("SignDict Logo") %>" class="so-navigation--logo"/></a>
-        <a href='/about' class='so-navigation--about'><%= gettext("About SignDict")%></a>
+<header class="so-navigation">
+  <div class="so-navigation__left">
+    <%= if assigns[:logo] == nil || assigns[:logo] == true do %>
+      <a href="/"><img src='/images/logo.svg' alt="<%= gettext("SignDict Logo") %>" class="so-navigation__logo"/></a>
+      <a href='/about' class='so-navigation__about'><%= gettext("About SignDict")%></a>
+    <% else %>
+      <a href='/about' class='so-navigation__about--no-logo'><%= gettext("About SignDict")%></a>
+    <% end %>
+  </div>
+
+  <nav role="navigation">
+    <input class="so-navigation__menu-btn" hidden type="checkbox" id="so-navigation__menu-btn" />
+    <label class="so-navigation__menu-icon" for="so-navigation__menu-btn">
+      <span class="so-navigation__navicon"></span>
+    </label>
+
+    <%= if assigns[:logo] == nil || assigns[:logo] == true do %>
+      <ul class="so-navigation__menu">
+    <% else %>
+      <ul class="so-navigation__menu so-navigation__menu--no-logo">
+    <% end %>
+
+      <%= if @current_user do %>
+        <%= if Canada.Can.can? @current_user, :show_backend, %{} do %>
+          <li>
+            <%= link "Admin", to: backend_dashboard_path(@conn, :index), data: [turbolinks: "false"] %>
+          </li>
+        <% end %>
+
+        <li>
+          <%= form_tag(session_path(@conn, :delete, @current_user), method: :delete) %>
+            <input type="submit" value="<%= gettext("Sign out") %>" class='sc-link so-navigation__menu-item--logout' />
+          </form>
+        </li>
+
+        <li>
+        <%= if Gettext.get_locale(SignDict.Gettext) == "de" do %>
+          <a href="?locale=en" data-turbolinks="false">English</a>
+        <% else %>
+          <a href="?locale=de" data-turbolinks="false">Deutsch</a>
+        <% end %>
+        </li>
+
+        <li class="so-navigation__menu-item--user">
+          <a href="<%= user_path(@conn, :show, @current_user) %>"><%= img_tag SignDict.User.avatar_url(@current_user), alt: gettext("The avatar image of this user"), class: "so-navigation__user-avatar" %><span class="so-navigation__user-name"><%= @current_user.name %></span></a>
```

Comment:
(No) White space matters here. 

Instead of showing the user's name, we could display 'My profile' or similar.